src/text/3-applications/3.4-survey_pops.tex

```diff
@@ -0,0 +1,8 @@
+\subsection{Surveys, catalogs, and population studies}
+
```

Comment:
Thanks s lot @QRemy! Just a first general comment: I think the whole paragraph could be written slightly more from a technical / modeling / performance perspective and maybe advertise a bit more what Gammapy is capable of. Some randoms thoughts:
- Mention typical exposure times, image sizes, number of pixels, energy bins, etc
- Mention order of sources to be modeled
- Mention various models / morphologies to be tested
- Seed catalog creation by TS maps, simple interface to 3rd party packages such as scikit-image
- We could add an image, but probably the references to the existing studies are sufficient

Reply:
For complex applications I had in mind to simply refers to exiting studies that used gammay or that present methods that we implemented in gammy so we can keep it short. But I agree what I wrote so far is not enough specific on how gammapy is used in these studies.
Regarding the points listed :
 - Mention typical exposure times, image sizes, number of pixels, energy bins, etc : depends on the instrument so we probably don't want to go this far in details.
 - Mention order of sources to be modeled : could mention that the order is important but I don't think there is an agreement on the best default model to be tested and the order.
 - Mention various models / morphologies to be tested : yes maybe also mention that the models are usually oversimplistic compared to the morphology of the extendend sources so they often end up modeled by multiple components.
- Seed catalog creation by TS maps, simple interface to 3rd party packages such as scikit-image :  In #25 there is a mention of Source Detections toward GC with Fermi-LAT, that's not the best region to show with this instrument (given the complexity/resolution). I could show a minimal example  doing this with the CTA-DC1 data including the hysteresis filtering of the significance maps (and eventually edge detection) with scikit-image.
- We could add an image, but probably the references to the existing studies are sufficient: just reference if we want to it short